docs/svg2-changelog.md

```diff
@@ -146,7 +146,7 @@ NOTE: This list is not final. This just things I was able to find so far. Patche
 
 - [x] [`textPath`](https://www.w3.org/TR/SVG2/text.html#TextPathElement) can reference [basic shapes](https://www.w3.org/TR/SVG2/shapes.html) now.
 - [ ] Since CSS Fonts Module Level 4, the [`font-weight`](https://www.w3.org/TR/css-fonts-4/#font-weight-prop) property allows any value in a 1..1000 range.
-- [ ] A [`writing-mode`](https://www.w3.org/TR/SVG2/text.html#WritingModeProperty) property has a new set of allowed values.
+- [x] The [`writing-mode`](https://www.w3.org/TR/SVG2/text.html#WritingModeProperty) property introduces the `horizontal-tb` and `vertical-lr` values from [CSS Writing Modes Level 3](https://www.w3.org/TR/css-writing-modes-3/#svg-writing-mode-css).
```

Comment:
Keep `[x] A`, since this is the article I use everywhere (not sure correctly or not).

Reply:
Done